README.md

```diff
@@ -2142,6 +2143,81 @@ Happy New Moon with Report is an open-source implementation of WebAssembly writt
     </tr>
     </table>
 
+
+
+## <a name="wazero"></a>[wazero](https://github.com/tetratelabs/wazero) <sup>[top⇈](#contents)</sup>
+> wazero is a WebAssembly 1.0 spec compliant runtime written in Go, with zero platform dependencies.
+
+* **Languages written in**
+
+    <table>
+    <tr>
+        <td>Go</td>
+    </tr>
+    </table>
+
+* **Compiler framework**
+
+    <table>
+    <tr>
+        <td>Custom</td>
+    </tr>
+    </table>
+
+
+* **Compilation / Execution modes**
+
+    <table>
+    <tr>
+        <td>Interpreted</td>
+        <td>JIT</td>
+    </tr>
+    </table>
+
+* **Interoperability with other languages**
+
+    - `N/A`
+
+* **Non-MVP features supported**
+
+    <table>
+    <tr>
+        <td>Import/Export of Mutable Globals</td>
+        <td>Sign-extension operators</td>
+        <td>Multi-Value Returns</td>
```

Comment:
fyi nontrapping-float-to-int-conversions is in [polishing stage](https://github.com/tetratelabs/wazero/pull/490) to be merged in the next few days. bulk-memory-operations is just getting started, but will land next.

Reply:
nontrapping-float-to-int-conversions is done!

Reply:
Added the nontrapping float-to-int conversions to the table. Guess I'll add Bulk Memory operations when those are ready?

Reply:
fyi the incredible @mathetake just finished bulk ops, so I guess this one is good to go. We're still iterating fast so more features will land soon!

Reply:
ps here's maybe a nicer chart to refer to https://github.com/tetratelabs/wazero/issues/484

Reply:
only thing left for wazero :heart: WebAssembly 2.0 is SIMD (aka Vector Instructions)